Resources/Public/Css/backend.css

```diff
@@ -1,34 +1,37 @@
 @charset "UTF-8";
 
-.t3-cd-badge-container {
-    position: relative;
+.t3js-page-column .t3-cd-badge-container[data-maxitems="0"] .t3-page-ce .t3js-page-new-ce,
```

Comment:
No, the Line 3-12 just removes the «new content-element» Button not the badge. 
Line 8-12 is a fix, to prevent the shifting of the Drop area if the button is hidden. 

The badge is always visible. We just hide it if there is no maxitems property in the backendlayout configuration.